sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -0,0 +1,250 @@
+
+package com.azure.ai.textanalytics;
+
+import com.azure.ai.textanalytics.models.AnalyzeSentimentResult;
+import com.azure.ai.textanalytics.models.DetectLanguageInput;
+import com.azure.ai.textanalytics.models.DetectLanguageResult;
+import com.azure.ai.textanalytics.models.DetectedLanguage;
+import com.azure.ai.textanalytics.models.DocumentResultCollection;
+import com.azure.ai.textanalytics.models.ExtractKeyPhraseResult;
+import com.azure.ai.textanalytics.models.LinkedEntity;
+import com.azure.ai.textanalytics.models.LinkedEntityMatch;
+import com.azure.ai.textanalytics.models.NamedEntity;
+import com.azure.ai.textanalytics.models.RecognizeEntitiesResult;
+import com.azure.ai.textanalytics.models.RecognizeLinkedEntitiesResult;
+import com.azure.ai.textanalytics.models.RecognizePiiEntitiesResult;
+import com.azure.ai.textanalytics.models.TextDocumentBatchStatistics;
+import com.azure.ai.textanalytics.models.TextDocumentInput;
+import com.azure.ai.textanalytics.models.TextDocumentStatistics;
+import com.azure.ai.textanalytics.models.TextSentiment;
+import com.azure.ai.textanalytics.models.TextSentimentClass;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Contains helper methods for generating inputs for test methods
+ */
+final class TestUtils {
+    private static final String DEFAULT_MODEL_VERSION = "2019-10-01";
+    static final List<String> SENTIMENT_INPUTS = Arrays.asList("The hotel was dark and unclean. The restaurant had amazing gnocchi.",
+        "The restaurant had amazing gnocchi. The hotel was dark and unclean.");
+
+    static final List<String> NAMED_ENTITY_INPUTS = Arrays.asList(
+        "I had a wonderful trip to Seattle last week.", "I work at Microsoft.");
+
+    static final List<String> LINKED_ENTITY_INPUTS = Arrays.asList(
+        "I had a wonderful trip to Seattle last week.",
+        "I work at Microsoft.");
+
+    static final List<String> KEY_PHRASE_INPUTS = Arrays.asList(
+        "Hello world. This is some input text that I love.",
+        "Bonjour tout le monde");
+
+    static final List<String> PII_ENTITY_INPUTS = Arrays.asList(
+        "Microsoft employee with ssn 859-98-0987 is using our awesome API's.",
+        "Your ABA number - 111000025 - is the first 9 digits in the lower left hand corner of your personal check.");
+
+    static final List<String> DETECT_LANGUAGE_INPUTS = Arrays.asList(
+        "This is written in English", "Este es un document escrito en Español.", "~@!~:)");
+
+    static List<DetectLanguageInput> getDetectLanguageInputs() {
+        return Arrays.asList(
+            new DetectLanguageInput("0", DETECT_LANGUAGE_INPUTS.get(0), "US"),
+            new DetectLanguageInput("1", DETECT_LANGUAGE_INPUTS.get(1)),
+            new DetectLanguageInput("2", DETECT_LANGUAGE_INPUTS.get(2), "US")
+        );
+    }
+
+    static List<TextDocumentInput> getTextDocumentNamedEntityInputs() {
+        return Arrays.asList(
```

Comment:
This will be cumbersome to manage if you add another "named_entity_input" or add to those other lists. I would suggest mapping.

```java
        return IntStream.range(0, contents.size())
            .mapToObj(index -> {
                return new TextDocumentInput(String.valueOf(index), contents.get(index));
            }).collect(Collectors.toUnmodifiableList());
```

Reply:
Updated